src/TLuaInterpreter.cpp

```diff
@@ -7844,6 +7844,16 @@ int TLuaInterpreter::addCustomLine(lua_State* L)
             }
             lua_pop(L, 1);
         }
+        if (!i || !x.count()) {
+            // If there is only an empty sub-table inside the table then i is
+            // one but there is nothing in any of the QLists and things will
+            // still blow up as per Issue #5272 - so also check for at least one
+            // x-coordinate value:
+            return warnArgumentValue(L, __func__, "no coordinate of points for the custom line given in table as second argument; at least one set of coordinates as a sub-table is required");
```

Comment:
```suggestion
            return warnArgumentValue(L, __func__, "missing coordinates to create the line to");
```

Reply:
I do not think that suggestion for the message gives *enough* detail, how about:
```suggestion
            return warnArgumentValue(L, __func__, "no custom line points given; at least one set (in an inner table) of coordinates as the second argument is needed");
```
which is a *bit* shorter?

Reply:
The opinion is with the first proposal... :grinning:

Reply:
Sod it, I can't be arsed to fight this - though it does go against my general feelings that the error messages should give some idea about what is needed to solve the problem without having to go and read up on it on-line...